Pages/Feedback.cshtml

```diff
@@ -19,7 +19,7 @@
 <section class="contact_section layout_padding">
     <div class="contact_bg_box">
         <div class="img-box">
-            <img src="images/contact-bg.jpg" alt="">
+            <img src="images/contact-bg.jpg" alt="contact" id="contact">
```

Comment:
alt text should describe the image, for someone who can't see the image.  What colors are used?  What is in the image?  How big is the image?

Is "id" used anywhere?  If not, it's excessive, and should be removed.